app/models/cloud_object_store_container.rb

```diff
@@ -17,7 +17,6 @@ class CloudObjectStoreContainer < ApplicationRecord
   alias_attribute :name, :key
 
   supports_not :delete, :reason => N_("Delete operation is not supported.")
```

Comment:
I would argue that this is a meaningless reason that can be auto-generated.   Thus, this support_not can be removed as well.